### DISCUSSION TITLE
Bot will stop playing audio if left to play a non-ending music stream/playlist.

### DISCUSSION THREAD

**Discussion by ghost** [eritislami/evobot#1272] [Q&A] [answered]
Essentially ITT, for instance - I would turn on a 24/7 livestream of music, after which a certain period of time elapses (seems to be over a day, but not really sure since I dont time when it is and is not playing audio for hours), the bot will stop playing the audio until you make it leave and rejoin. This is an annoying issue as it prevents me from being able to have the bot function as a 24/7 version of some audio (e.x a 24/7 Jazz VC) if it stops working after some random amount of time.

 Is there a fix?

**Comment by eritislami** [comment, marked as answer]
It's not designed to be a 24/7 or livestream bot, you will probably have better luck with some other bots
